crates/ruff_server/src/fix.rs

```diff
@@ -27,15 +27,14 @@ pub(crate) fn fix_all(
     encoding: PositionEncoding,
 ) -> crate::Result<Fixes> {
     let source_kind = query.make_source_kind();
-
-    let file_resolver_settings = query.settings().file_resolver();
+    let settings = query.settings();
     let document_path = query.file_path();
 
     // If the document is excluded, return an empty list of fixes.
     let package = if let Some(document_path) = document_path.as_ref() {
         if is_document_excluded_for_linting(
             document_path,
-            file_resolver_settings,
+            &settings.file_resolver,
             linter_settings,
             query.text_document_language_id(),
```

Comment:
I'm still passing these individually to `is_document_excluded_*` functions because some paths require a different linter settings (e.g., organize imports uses a specific settings which only selects `I`).